SourceEditor/app/src/main/java/com/microsoft/device/display/samples/sourceeditor/viewmodel/ScrollState.kt

```diff
@@ -0,0 +1,13 @@
+/*
+ *
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License.
+ *
+ */
+
+package com.microsoft.device.display.samples.sourceeditor.viewmodel
+
+/* Class used to define a key, value pair for scrolling data */
+data class ScrollState(val scrollKey: String, val scrollPercentage: Int) {
+    // empty body
+}
```

Comment:
there is no need to use curly braces with empty body in a data class this:
`data class ScrollState(val scrollKey: String, val scrollPercentage: Int) ` works too :-)